libraries/AP_GPS/AP_GPS_GSOF.cpp

```diff
@@ -71,17 +71,29 @@ AP_GPS_GSOF::AP_GPS_GSOF(AP_GPS &_gps, AP_GPS::GPS_State &_state,
 
     const uint32_t now = AP_HAL::millis();
     gsofmsg_time = now + 110;
+
+    const auto raw_data = gps._raw_data.get();
+    if(!validate_raw_data(raw_data)) {
+        GCS_SEND_TEXT(MAV_SEVERITY_ERROR, "GPS_RAW_DATA has invalid value %d", raw_data);
```

Comment:
This will spam the logs with error messages for invalid packets.

Reply:
> This will spam the logs with error messages for invalid packets.
I removed validating raw_data now, since none of AP_GPS_SBF or AP_GPS_Ublox were doing it.